demos/card.html

```diff
@@ -127,210 +38,226 @@
-              <h2 class="mdc-card__subtitle">Subhead</h2>
-            </section>
-            <section class="mdc-card__media demo-card__16-9-media"></section>
-            <section class="mdc-card__actions mdc-card__actions--vertical">
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 1</button>
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 2</button>
-            </section>
-          </div>
-        </div>
-
-        <div>
-          <div class="mdc-card demo-card">
-            <section class="mdc-card__media demo-card__16-9-media"></section>
-            <section class="mdc-card__primary">
-              <h1 class="mdc-card__title mdc-card__title--large">Title goes here</h1>
-              <h2 class="mdc-card__subtitle">Subtitle here</h2>
-            </section>
-            <section class="mdc-card__actions">
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 1</button>
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 2</button>
-            </section>
-          </div>
-        </div>
-
-        <div>
-          <div class="mdc-card demo-card">
-            <section class="mdc-card__primary">
-              <h1 class="mdc-card__title mdc-card__title--large">Title goes here</h1>
-              <h2 class="mdc-card__subtitle">Subtitle here</h2>
-            </section>
-            <section class="mdc-card__supporting-text">
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-              Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-            </section>
-            <section class="mdc-card__actions">
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 1</button>
-              <button class="mdc-button mdc-button--compact mdc-card__action">Action 2</button>
-            </section>
+          <div class="mdc-card__actions">
+            <div class="mdc-card__action-buttons">
+              <button class="mdc-button mdc-card__action">Read</button>
+              <button class="mdc-button mdc-card__action">Bookmark</button>
+            </div>
+            <div class="mdc-card__action-icons">
+              <i class="mdc-icon-toggle material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 aria-pressed="false"
+                 aria-label="Add to favorites"
+                 title="Add to favorites"
+                 data-toggle-on='{"content": "favorite", "label": "Remove from favorites"}'
+                 data-toggle-off='{"content": "favorite_border", "label": "Add to favorites"}'>
+                favorite_border
+              </i>
+              <button class="mdc-button mdc-card__action mdc-card__action--icon" title="Share">
```

Comment:
Removed the `mdc-button` wrappers. Now they're just plain icons with unbounded ripples.

I was originally going to add `overflow: hidden` to `mdc-card`, but that would break overflow menus, selects, and probably lots more stuff in the future (e.g., tooltips).

Also, it looks weird when the icon toggle ripple gets truncated by the card.